app/javascript/bundles/RestaurantsApp/components/RestaurantsApp.jsx

```diff
@@ -118,6 +181,33 @@ export default class RestaurantsApp extends React.Component {
         });
     }
 
+
+    clearFilters() {
+        if (this.state.filtersApplied) {
+            this.setState({
+                restaurantFilter: "",
+                minRating: null,
+                cuisineTypeFilter: null,
+                maxSpeed: 0,
+                is10bisFilter: false,
+                isKosherFilter: false
+            }, this.filterRestaurants);
+        }
+    }
+
+    areFiltersApplied() {
+        const noFilters = (
+            (this.state.restaurantFilter === null || this.state.restaurantFilter.length === 0) &&
+            (this.state.minRating === null || this.state.minRating === 0) &&
+            (this.state.cuisineTypeFilter === null || this.state.cuisineTypeFilter.length === 0) &&
+            (this.state.maxSpeed === 0) &&
+            (!this.state.is10bisFilter) &&
+            (!this.state.isKosherFilter)
+        );
+
+        return !noFilters;
+    }
```

Comment:
I don't know what would be the best way to implement this, but looks to me like this could be written in a more scalable manner (think of, for example, a map of filter name to function, for example), and an `activeFilters` object.
See @yonidor's comment here: https://github.com/NoyaArie/AppWeEat/pull/5#discussion_r143509363

We can talk about possible implementations if you'd like, but you can consider this out of scope.